CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+
+# CHANGELOG
+
+## v3.2.0
```

Comment:
a.) @rgallettonf needs to be ok with bumping this version
b.) does it really need to be bumped just to add a new image?
c.) i'm still not liking the confustion `ziti-console` creates. i'd like to see any sort of qualifier on it. -spa, -partial, -something aything to give people a hint that this probably is not the "full thing" and it's a "baked" image. there must be some kinda pattern in place for images like this that are "not full and complete"?

Reply:
I put links to the deployment guide in [the image's Docker Hub repo](https://hub.docker.com/r/openziti/ziti-console) to prevent someone from trying to run `openziti/ziti-console` directly. Also, that image prints "see deployment guide" with the link if they do run it. 

```bash
❯ docker run --rm openziti/ziti-console:3.1.0 
deployment guide https://openziti.io/docs/guides/deployments/docker/console
```

These measures mitigate the risk of confusion about the purpose of the new image `openziti/ziti-console` because the most likely point of entry for a confused user is Docker Hub, and the solution is to follow the deployment guide which clarifies that the container image does not run any application, but provides the static files that must be hosted by webserver.

I've had the thought that we could generalize this solution in the future for all webservers, and no longer require that the ziti controller host the static files, but I don't know enough about how the SPA works to say whether that's viable.